### ISSUE TITLE
Antivirus removed eicar.txt after clone

### ISSUE THREAD

**Issue by xiegeo** [baruwa-enterprise/clamd#1]
I cloned the repo in Windows 10 with Defender active, which removed the file eicar.txt automatically.

I think modifying the file, such as rot13 or base64 encode, can hide the test file from AVs. If this the appropriate action, I would like to send a PR with updated testing code.

**Comment by akissa** [comment]
I do not see how that would work as even if you recreate the file, defender will still delete the recreated file before the test runs.

**Comment by xiegeo** [comment]
Can we use Client.ScanReader instead of Client.InStream? Or is that path part of the test coverage?

I think file creation is a little better than having it in the repo, the test could be marked as skipped if the file got deleted.

**Comment by akissa** [comment]
Yes it is part of the test coverage. You can send a pull request with the skip.

**Comment by xiegeo** [comment]
With the original eicar.txt unmodified or create a new one during testing (from data embedded in the test or another method)?

If the original file stays, I'd prefer for the tests to fail. Since removing a committed file is a serious anomaly.
